sdk/core/core/test/cmocka/test_az_encode.c

```diff
@@ -59,3 +66,11 @@ void test_url_encode(void** state)
     assert_true(az_span_is_content_equal(az_span_slice(builder, 0, 636), uri_encoded));
   }
 }
+
+int test_az_url_encode()
```

Comment:
We shouldn't have renamed this file :(

https://github.com/Azure/azure-sdk-for-c/pull/562#discussion_r406495675

Reply:
?
The comment was asking to rename it from test_az_encode to test_az_url_enconde..  
It's not moved to az_span tests becase it has a lot a test data that we don't want to add in az_span test